portfolio/src/main/java/com/google/sps/servlets/CommentsListServlet.java

```diff
@@ -31,38 +32,36 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.time.format.DateTimeFormatter;
-import java.time.LocalDateTime;  
-
+import java.time.LocalDateTime;
 
 /** Servlet responsible for listing comments. */
 @WebServlet("/comments-list")
 public class CommentsListServlet extends HttpServlet {
-  
+
   private DatastoreService datastore;
-  
+
   @Override
-  public void init(){
+  public void init() {
     datastore = DatastoreServiceFactory.getDatastoreService();
-  } 
-  
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    int maxComments = Integer.parseInt(request.getParameter("max")); 
+    int pageNumber = Integer.parseInt(request.getParameter("pageNumber"));
+    int maxComments = 3; 
```

Comment:
Naming for this would typically be 
```java
final int kMaxComments = 3;
```

The "kName" is a google-wide naming schema for constants. I don't know what "k" stands for... maybe "konstant"?

Reply:
Sorry, but could you reverse this change (back to MAX_COMMENTS). I checked the style guide (and confirmed with Alex), that `kVariableName` is not the formatting for Java, just C++.